rust/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM rust:1.57
```

Comment:
Looks good would be great to add an entry to `base-compose.yml` https://github.com/CoderOneHQ/bomberland/blob/master/base-compose.yml

Reply:
@thegalah Done!